test/components/general/OrganizationSelect.test.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import OrganizationSelect from "../../../src/components/general/OrganizationSelect";
+import renderer from "react-test-renderer";
+
+describe("OrganizationSelect component", () => {
+  let organizations = [
+    {
+      name: "sneep Erlangen e.V.",
+      organization_image: "tmp-data/assets/images/sneep.png",
+      id: 44
+    },
+    {
+      name: "Climate Action Tech",
+      organization_image: "tmp-data/assets/images/climate-action-tech.png",
+      id: 45
+    }
+  ];
+  it("renders each organization as a MenuItem", () => {
```

Comment:
I'm new to React snapshot testing. Is this the kind of spec title that is used? It seems like we could completely reorganize the component to not render `MenuItem`s and the test could still pass if we updated the snapshot.